.github/workflows/test-lms-harmonizer.yml

```diff
@@ -0,0 +1,65 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+name: LMS Harmonizer - Test
+on:
+  pull_request:
+    paths:
+      - 'src/lms-harmonizer/**'
+      - '!src/lms-harmonizer/**/*.md'
+  push:
+    paths:
+      - 'src/lms-harmonizer/**'
+      - '!src/lms-harmonizer/**/*.md'
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  test-canvas-extractor:
+    name: Run unit, style, and type checks
+    runs-on: ubuntu-20.04
+    env:
+        PACKAGE_NAME: lms-harmonizer
+        SRC_DIR: src/lms-harmonizer
+        LOCK_FILE: src/lms-harmonizer/poetry.lock
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+
+      - name: Install Python 3.9
+        uses: actions/setup-python@3105fb18c05ddd93efea5f9e0bef7a03a6e9e7df
+        with:
+          # This one needs to be hard-coded to avoid 3.9.5, due to a bug
+          # in one of the dependencies
+          python-version: '3.9.4'
```

Comment:
Harmonizer doesn't depend on very much. I bet it will work with `3.9.x`, unlike Canvas. Please try it out.

Reply:
Also, this should be consistent with the python version range in the pyproject.toml

Reply:
✔️